CaImAn/caiman_run.py

```diff
@@ -9,15 +9,15 @@
 
 # Import for CaimAn lib
 import caiman as cm
-from caiman import stop_server
 from caiman.cluster import setup_cluster
 from caiman.source_extraction import cnmf
 from caiman.source_extraction.cnmf import params
 from caiman.motion_correction import MotionCorrect
 
 # Import CaimAn related utilities libraries
-from utilities import get_frequency, get_dims, load_attributes, save_attributes, print_to_intercept
-from caiman_utilities import save_caiman_to_doric, set_advanced_parameters
+import utilities as utils
+from utilities import print_to_intercept
```

Comment:
If import as utils above, why not using it for this function as well?

Reply:
Just so I don't have to do utils.print_to_intercept(....) every time. I found it a bit cumbersome in this case compare to the other ones.

Also because I'm using it a lot in minian I wanted to be the same here